src/index.html

```diff
@@ -11,6 +11,57 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <section class="search">
+      <form
+        action="#"
+        method="post"
+        data-qa="big"
+      >
+        <div class="search__block search__block-big">
+          <img
+            class="img"
+            src="images/Search.svg"
+            alt="Search"
+          >
+          <label
+            for="big-input"
+            class="label"  
+          >
+            <input
+              id="big-input"
+              class="input"
+              data-qa="keypress"
+              type="text"
+              placeholder="Try &quot;Los Angeles&quot;"
+            >
+          </label>
+        </div>
+      </form>
+      <form
```

Comment:
Don't forget to add empty lines between multilane children
```suggestion
      </form>
      
      <form
```